fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -4010,125 +4010,8 @@
             <Component ID="Audio_Panel_Left">
                 <UseTemplate Name="FBW_AIRLINER_Audio_Panel_Template">
                     <ID>1</ID>
+                    <PANEL_FOR_ANIM>AUDIOL</PANEL_FOR_ANIM>
                     <SIDE>L</SIDE>
-                    <NODE_ID_RECEIVER_VHF_L>KNOB_AUDIOL_VHF1</NODE_ID_RECEIVER_VHF_L>
```

Comment:
The idea here was for the logic templates to be decoupled from the 3d model names, so we can re-use the logic with multiple different 3D models etc. Now we'd have to duplicate all the logic.

Reply:
Ok I didn't know that. Will revert once home

Reply:
Done